docs/advance.rst

```diff
@@ -110,6 +110,80 @@ This example show how to specify default AWS S3 parameters, a `LocalStack`_ Buck
    >>> register_configuration_parameter(minio_bucket_path, resource=minio_resource)
 
 
+while using rename to/from S3 path we are using upload/download methods
+but s3 transfer configuration set in your `.aws/config` file won't be honored
+as this is not part of boto3_ but config interpretation is part of `aws-cli`_ in the
+`RuntimeConfig` class.
+
```

Comment:
I've to warn that rename from/to fs support depends on uri-pathlib-factory  library